guides/common/modules/con_host-group-settings-and-nested-host-groups.adoc

```diff
@@ -1,7 +1,7 @@
 [id="Host_Group_Settings_and_Nested_Host_Groups_{context}"]
 = Host group settings and nested host groups
 
-A host group can define many settings for hosts, such as Lifecycle Environment, Content View, or Ansible roles that are available to the hosts.
+A host group can define many settings for hosts, such as Lifecycle Environment, content view, or Ansible roles that are available to the hosts.
```

Comment:
Seeing this, I wonder if lifecycle environment is another term which we need to fix the capitalization for. Opinions?

Reply:
+1. This would have to be in a separate PR.